gems/pending/metadata/linux/LinuxUtils.rb

```diff
@@ -141,6 +141,44 @@ def self.parse_systemctl_list(lines)
       end.compact!
     end
 
+    def self.collect_interface(interfaces, interface)
+      mac_addr = interface[:mac_address]
+      return if mac_addr.blank?
+
+      existing_interface = interfaces[mac_addr]
+      if existing_interface.blank?
+        interfaces[mac_addr] = interface
+      else
+        interface[:name] += "," + existing_interface[:name]
+        existing_interface.merge!(interface)
+      end
+    end
+
+    def self.parse_network_interface_list(lines)
```

Comment:
This feels very specific to a single kind of system.  If the format is different, how can you ensure this will work correctly.  I feel like this would be better served by living something like LinuxAdmin, where there could be a helper method to parse the output based on the format of the text. cc @carbonin Thoughts here?